app/controllers/checkout_controller.rb

```diff
@@ -0,0 +1,6 @@
+class CheckoutController < ApplicationController
+  def index
+    @name = params[:name]
```

Comment:
Does the `@name` belong to a checkout? If so, what is an example of a checkout name?